styles/styles.css

```
@@ -180,6 +243,12 @@ main {
     background-color: rgba(131, 179, 211, 0.2);
 }
 
+.advantages-service-item-first,
+.advantages-service-item-secound,
+.advantages-service-item-third {
+    width: 400px;
+}
+
```

```suggestion
.advantages-service-item {
    width: 400px;
}

```
Удобно дать какой-то общий класс.
Например, `advantages-service-item`